Add schema for dictionary

theschitz opened the pull request:
<!-- If there's an open issue please reference this here. -->
Fixes #237  .

<!-- If there's no open issue consider opening one first otherwise please describe the changes. -->
Changes proposed in this pull request:

- Adds schema for DTS dictionary.

Additional comments
Went with the enum for language code, it's either that or regex. Enum felt more helpful to the user and could give us a heads up if we're missing a base app translation map.

jwikman commented:
I don't know if we really should go for the enums for languages. Partners can submit their own languages nowadays, so we will probably not support all languages with this approach...

theschitz commented:
> I don't know if we really should go for the enums for languages. Partners can submit their own languages nowadays, so we will probably not support all languages with this approach...

Yeah, I had this idea that we could add them as they were found by users. But since the dictionary is auto generated it would probably not be very common for the user to touch that and it would be more of a nuisance than help. 

Removed enums and updated description.

jwikman reviewed (approved):
It's hard to review without testing it manually, but it's looking good!

So proceed and merge this and I'll do a manual test run before pushing out a pre-release.
